packages/widget/src/utils/index.ts

```diff
@@ -84,3 +84,16 @@ export const validateAddress = (
       return 'unsupported network';
   }
 };
+
+export const debounceAmountChange = <Args extends string>(
```

Comment:
does this need to be generic since it's only used for amount change?

maybe it'd be better to put just `T` instead of `Args extends string` and rename it to `debounce`? this way it can be reused for any function that needs debouncing